common-utils/src/main/java/eu/arrowhead/common/intf/properties/validators/NotEmptyStringSetValidator.java

```diff
@@ -45,17 +45,17 @@ public Object validateAndNormalize(final Object propertyValue, final String... a
 			}
 
 			final Set<String> normalized = new HashSet<>(set.size());
-			final boolean isName = (args.length > 0 ? args[0] : "").trim().equalsIgnoreCase(nameArg);
+			final boolean isName = (args.length > 0 ? args[0] : "").trim().equalsIgnoreCase(ARG_NAME);
 			for (final Object object : set) {
 				if (object instanceof final String str) {
 					if (isName) {
 						nameValidator.validateName(str.trim());
 						normalized.add(nameNormalizer.normalize(str));
 					} else {
-						final String trimmed = str.trim();
-						if (Utilities.isEmpty(trimmed)) {
+						if (Utilities.isEmpty(str)) {
```

Comment:
Why is it better? I think we should check if the trimmed string is empty. Maybe `if (Utilities.isEmpty(str.trim()))` ?